src/server.ts

```diff
@@ -29,17 +27,29 @@ setInterval(function () {
     .then(function () {
       // always executed
     });
-}, 60 * 1000);
+}, 5 * 1000);
```

Comment:
let's keep 1 minute interval :)

Reply:
My bad, forget to rollback this line, now I use 1 minute again